src/librustc_trans/trans/context.rs

```diff
@@ -838,7 +838,9 @@ fn declare_intrinsic(ccx: &CrateContext, key: &str) -> Option<ValueRef> {
 
     ifn!("llvm.trap", fn() -> void);
     ifn!("llvm.debugtrap", fn() -> void);
-    ifn!("llvm.frameaddress", fn(t_i32) -> i8p);
+    if !ccx.sess().target.target.options.is_like_pnacl {
```

Comment:
It looks like this is the only use case of `is_like_pnacl`, and it also looks like `llvm.frameaddress` isn't used at all, so perhaps this could just be removed entirely?